src/vmm_unix.ml

```diff
@@ -201,10 +201,10 @@ let destroy_tap tap =
 let cachet_of_str b =
   let map () ~pos len =
     if pos >= String.length b || len <= 0 then
-      (Cachet.Bstr.empty :> Cachet.bigstring)
+      (Cachet.Bstr.empty :> Bstr.t)
     else
       let len = min len (max 0 (String.length b - pos)) in
-      let pg : Cachet.bigstring = Bigarray.Array1.create Bigarray.char Bigarray.c_layout len in
```

Comment:
We can also use `Bstr.create len` now if you want.

Reply:
where? in the line above, `let len = ...`?